src/deluge/io/midi/midi_device_manager.h

```diff
@@ -30,10 +30,14 @@ struct MIDIDeviceUSB;
 #endif
 
 // size in 32-bit messages
-// TODO: increasing this even more doesn't work. For now this gives
-// maximum SysEx send size of 96 bytes including start/end bytes
-// (3 payload bytes per USB-MIDI message)
-#define MIDI_SEND_BUFFER_LEN 32
+// NOTE: increasing this even more doesn't work.
+// Looks like a hardware limitation (maybe we more in FS mode)?
```

Comment:
What happens?

Reply:
I've just been reading the the usb midi spec and we might be able to use the maximum buffer size specified by a downstream device instead of this guesswork.